workbooks/models.py

```diff
@@ -66,10 +66,6 @@ def create(cls, name, description, user, build='HG38'):
     @classmethod
     def createDefault(cls, name, description, user):
         workbook_model = cls.create(name, description, user)
-        worksheet_model = Worksheet.objects.create(name="worksheet 1",
-                                                   description="",
-                                                   workbook=workbook_model)
-
         return workbook_model
```

Comment:
This breaks the code because it halts creation of the worksheet. The variable can be unassigned, but the method still needs to be called.